frontend/src/app/store/reducers/cases.ts

```diff
@@ -445,4 +468,69 @@ const parseAssessmentResponse = async (res: CaseFileDto, officers: Officer[]): P
   }
 }
 
+export const createReview = (complaintId: string, isReviewRequired: boolean, reviewComplete: any ): AppThunk => async (dispatch, getState) => {
+  const {
+    app: { profile },
+    cases: { caseId }
+  } = getState();
+  let reviewInput = {
+    reviewInput: {
+      leadIdentifier: complaintId,
+      caseIdentifier: caseId, 
+      userId: profile.idir_username,
+      agencyCode: "COS",
+      caseCode: "HWCR",
+      isReviewRequired
+    } as any
```

Comment:
You shouldn't use any here, and instead you should create a type for this. You'll want to do the same thing in your update review as well